src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -75,3 +75,7 @@ ascanalpha.relativepathconfusion.extrainfo.quirksmodeenabledimplicitly=Quirks Mo
 ascanalpha.relativepathconfusion.extrainfo.quirksmodeenabledimplicitlynodoctype=Quirks Mode is implicitly enabled via the absence of a DOCTYPE, allowing the specified Content Type to be bypassed.
 ascanalpha.relativepathconfusion.extrainfo.nocontenttype=No Content Type was specified, so Quirks Mode is not required to exploit the vulnerability in the web browser. 
 ascanalpha.relativepathconfusion.extrainfo.framingallowed=No X-Frame-Options header was specified, so the page can be framed, and this can be used to enable Quirks Mode, allowing the specified Content Type to be bypassed.
+
+ascanalpha.testuseragent.name=User Agent Fuzzer
+ascanalpha.testuseragent.desc=Check for differences in content based on User Agent (eg, Mobile sites, access as a Search engine Crawler)
```

Comment:
Maybe "response" instead of "content"?
It should tell which differences is the scanner looking for, it's easier to assess/review the alert.